distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -96,7 +96,8 @@ parse_builtin text = @Builtin_Method "Date_Time.parse_builtin"
    the Europe/Paris timezone" can be stored as `Time`.
 @Builtin_Type
 type Date_Time
-    ## Obtains the start of the epoch for Enso.
+    ## PRIVATE
+       Obtains the start of the epoch for Enso.
```

Comment:
Why is this private?

Reply:
Made public again - was in two minds on this one.